source/common/upstream/load_stats_reporter.h

```diff
@@ -45,7 +45,7 @@ class LoadStatsReporter
   void onRemoteClose(Grpc::Status::GrpcStatus status, const std::string& message) override;
 
   // TODO(htuch): Make this configurable or some static.
-  const uint32_t RETRY_DELAY_MS = 5000;
+  const uint32_t RETRY_DELAY_MS = 30000;
```

Comment:
Please revert or make configurable.

Reply:
Ack.  Is a log change really high-risk enough to warrant a runtime guard though?

Reply:
It's more than a log change, right? The conditions under which a gRPC error are returned have changed and there's a change to the retry delay.

Reply:
I've reverted the retry delay.  The only difference in behavior at this point is that Grpc::Status::WellKnownGrpcStatus::Ok is logged as a debug instead of a warning, and we don't increment the error metric.

Reply:
Ah, I see. IMO this doesn't need it anymore, but I'll let @adisuissa decide if the flag is still necessary.

However, it does still need a release note.

Reply:
This was mainly about RETRY_DELAY_MS which was a user-facing change.